<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumArtManagerDialog</class>
 <widget class="QDialog" name="AlbumArtManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Album art manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="ArtistLine_">
     <property name="placeholderText">
      <string>Artist</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="AlbumLine_">
     <property name="placeholderText">
      <string>Album</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="ArtView_">
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="gridSize">
      <size>
       <width>250</width>
       <height>300</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BrowseButton_">
       <property name="text">
        <string>Browse...</string>
       </property>
       <property name="icon">
        <iconset theme="document-open"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>AlbumArtManagerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>AlbumArtManagerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
